embedded-tests/src/test/java/org/apache/druid/testing/embedded/auth/AllowAllWithPolicyAuthResource.java

```diff
@@ -0,0 +1,183 @@
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.testing.embedded.auth;
+
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.fasterxml.jackson.databind.Module;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.google.inject.Binder;
+import com.google.inject.Key;
+import com.google.inject.multibindings.MapBinder;
+import org.apache.druid.guice.LazySingleton;
+import org.apache.druid.guice.PolyBind;
+import org.apache.druid.initialization.DruidModule;
+import org.apache.druid.query.policy.NoRestrictionPolicy;
+import org.apache.druid.server.security.Access;
+import org.apache.druid.server.security.Action;
+import org.apache.druid.server.security.AllowAllAuthenticator;
+import org.apache.druid.server.security.AuthConfig;
+import org.apache.druid.server.security.AuthenticationResult;
+import org.apache.druid.server.security.Authenticator;
+import org.apache.druid.server.security.AuthorizationUtils;
+import org.apache.druid.server.security.Authorizer;
+import org.apache.druid.server.security.NoopEscalator;
+import org.apache.druid.server.security.Resource;
+import org.apache.druid.testing.embedded.EmbeddedDruidCluster;
+import org.apache.druid.testing.embedded.EmbeddedResource;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+public class AllowAllWithPolicyAuthResource implements EmbeddedResource
+{
+  private static final String AUTH_TYPE = "actually-allow-all";
+
+  @Override
+  public void start() throws Exception
+  {
+    // Do nothing
+  }
+
+  @Override
+  public void stop() throws Exception
+  {
+    // Do nothing
+  }
+
+  @Override
+  public void beforeStart(EmbeddedDruidCluster cluster)
+  {
+    cluster.addExtension(AllowAllWithPolicyAuthModule.class)
+           .addCommonProperty("druid.auth.authenticatorChain", "[\"test\"]")
+           .addCommonProperty("druid.auth.authenticator.test.type", "actually-allow-all")
+           .addCommonProperty("druid.auth.authorizers", "[\"test\"]")
+           .addCommonProperty("druid.auth.authorizer.test.type", "actually-allow-all")
```

Comment:
Instead of writing a new one, can't we use the `allowAll` authenticaator/authorizer policy itself alongwith `enforcer.allowedPolicies` config?